backend/src/plugins/Mutes/functions/unmuteUser.ts

```diff
@@ -54,8 +54,11 @@ export async function unmuteUser(
     }
 
     // Update timeout
-    if (existingMute?.type === MuteTypes.Timeout || createdMute?.type === MuteTypes.Timeout) {
-      await member?.disableCommunicationUntil(timeoutExpiresAt);
+    if (member && (existingMute?.type === MuteTypes.Timeout || createdMute?.type === MuteTypes.Timeout)) {
+      if (!member.manageable || !member.moderatable || member.permissions.has(PermissionsBitField.Flags.Administrator))
+        return null;
```

Comment:
All of this is already checked by `moderatable`
```suggestion
      if (!member.moderatable) return null;
```